src/main/java/org/jenkinsci/plugins/github_branch_source/PullRequestSCMHead.java

```diff
@@ -76,7 +76,7 @@
         this.metadata = copy.metadata;
     }
 
-    PullRequestSCMHead(GHPullRequest pr, String name, boolean merge) {
+    public PullRequestSCMHead(GHPullRequest pr, String name, boolean merge) {
```

Comment:
not this one

Reply:
I don't think I noticed `PullRequestSCMHead(@NonNull String name, String sourceOwner, String sourceRepo, String sourceBranch, int number, BranchSCMHead target, SCMHeadOrigin origin, ChangeRequestCheckoutStrategy strategy)` before but having it be the public one makes sense to me.